uuid/uuid_test.go

```diff
@@ -0,0 +1,136 @@
+package uuid_test
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	. "github.com/tarantool/go-tarantool"
+	_ "github.com/tarantool/go-tarantool/uuid" 
+	"gopkg.in/vmihailenco/msgpack.v2"
+	"github.com/google/uuid"
+)
+
+var server = "127.0.0.1:3013"
+var opts = Opts{
+	Timeout: 500 * time.Millisecond,
+	User:    "test",
+	Pass:    "test",
+	//Concurrency: 32,
```

Comment:
Remove the commented code or add a comment why it shouldn't be removed.

Reply:
Fixed